### PR TITLE
Added Landing Page

### PR DESCRIPTION
**Pull request by KenPrz**
Credits to : Kevs

- landing page added
- login buttons implemented (wip)
- css & js relocated

### CODE REVIEW

**Review by Teds30 — COMMENTED** [review]
paadd nalang ng route nito sa web.php

**Comment by ghost** [comment]
will do

**Comment by ghost** [comment]
ay maling account

**Comment by FN-FAL113** [comment]
merge ko to pag may routes na para ma test ko

**Comment by KenPrz** [comment]
fixed

File: routes/web.php [hunk 1]
```diff
@@ -21,7 +21,7 @@
 */
 
 
-Route::get('/', function () {
+Route::get('/index', function () {
```
Comment (Teds30) [inline routes/web.php]: remove 'index' nalang, keep '/'

File: routes/web.php [hunk 2]
```diff
@@ -77,18 +77,18 @@
 
 //guest info
 Route::get('/guestinfo/guest-form', function () {
-    return view('guest-information/Guest Information Form');
+    return view('guest-information.guest_information_form');
 });
 
 Route::get('/guestinfo/reservation', function () {
-    return view('guest-information/Reservation');
+    return view('guest-information.reservation');
```
Comment (Teds30) [inline routes/web.php]: walang 'reservation' na blade template, 'reservation_form' ata to

**Review by Teds30 — CHANGES_REQUESTED** [review]
pafix ng typos

File: routes/web.php (hunk 2 above)
Comment (KenPrz) [reply routes/web.php]: fixed

File: routes/web.php (hunk 1 above)
Comment (KenPrz) [reply routes/web.php]: done

**Review by Teds30 — APPROVED** [review]
working na mga routes

**Pull request merged**